server/server.sbt

```diff
@@ -3,7 +3,8 @@ name := "http4s-server"
 description := "Server bindings for http4s"
 
 libraryDependencies ++= Seq(
-  metricsCore
+  metricsCore,
+  "com.typesafe.akka" %% "akka-actor" % "2.3.9"
```

Comment:
Yes, I strongly wish to avoid an Akka dependency.  We already use scalaz actors in the non-blocking servlet I/O.